TransferMethodRepository/Sources/TransferMethodConfigurationRepository.swift

```diff
@@ -40,6 +40,17 @@ public protocol TransferMethodConfigurationRepository {
     /// - Parameter completion: the callback handler of responses from the Hyperwallet platform
     func getKeys(
         completion: @escaping (Result<HyperwalletTransferMethodConfigurationKey?, HyperwalletErrorType>) -> Void)
+    
+    ///  Retrieve the transfer method types and their fees and processing times based on the parameters
+    ///
+    /// - Parameters:
+    ///   - country: the 2 letter ISO 3166-1 country code
+    ///   - currency: the 3 letter ISO 4217-1 currency code
+    ///   - completion: the callback handler of responses from the Hyperwallet platform
+    func retrieveTransferMethodTypesFeesAndProcessingTimes(
```

Comment:
In Repository we normally use get instead of retrieve so follow the same naming convention.

Reply:
Renamed